### DISCUSSION TITLE
Problem with displaying letters in Postal

### DISCUSSION THREAD

**Discussion by Toni202012569** [postalserver/postal#2713] [Help with using Postal]
Good afternoon. Please tell me how to fix the problem with displaying letters when viewing through Postal. For some reason, instead of letters, I have only hieroglyphs.
![Скриншот 29-11-2023 13 08 58](https://github.com/postalserver/postal/assets/139015871/ad0374a3-cb5b-4fee-9982-4871950ccdb0)

**Comment by willpower232** [comment]
Have a look at https://github.com/postalserver/postal/issues/2494
